index.html

```diff
@@ -5,184 +5,285 @@
-        <ul class="container">
-            <li class="skills">
-                <h5>Languages</h5>
-                <img src="./img/down-arrow.png" alt="">
-            </li>
-            <li class="list list-item icon">
-                <img src="./img/js.png" alt="">
-                <p>JavaScript</p>
-            </li>
-            <li class="list list-item">
-                <img src="./img/html.png" alt="">
-                <p>HTML</p>
-            </li>
-            <li class="list list-item">
-                <img src="./img/css.png" alt="">
-                <p>CSS</p>
-            </li>
-            <li class="skills line">
-                <h5>Frameworks</h5>
-                <img src="./img/right-arrow.png" alt="">
-            </li>
-            <li class="skills line">
-                <h5>Skills</h5>
-                <img src="./img/right-arrow.png" alt="">
-            </li>
-        </ul>
+
     </section>
-    <div class="form-container">
-        <form action="https://formspree.io/f/xyyovkvo" method="post">
-            <h2>Contact me</h2>
-            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-            <input type="text" id="Name" name="Name" placeholder="Your name.." minlength="3" maxlength="30" required>
-            <input type="email" id="email" name="email" placeholder="  Your e-mail.." required>
-            <textarea id="message" name="subject" maxlength="500" placeholder="Write your message here" required></textarea>
-            <input type="submit" value="Get in touch">
-        </form>
-        <div class="endline">
+    <section id="contact" class="form-container">
+        <div class="form-min">
+            <div class="form-des">
+                <h1>Contact me</h1>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            </div>
+            <form method="POST" action="https://formspree.io/f/xyyovkvo">
+                <ul class="form-ul">
+                    <li>
+                        <input type="text" name="name" id="user-name" placeholder="Your name here.." required>
+                    </li>
+                    <li>
+                        <input type="email" name="email" id="email-address" placeholder="Your E-mail here.." required>
+                    </li>
+                    <li>
+                        <textarea name="message" id="msg" maxlength="500" placeholder="Write your Message Here" required></textarea>
+                    </li>
+                    <li>
+                        <button type="submit">Get In Touch</button>
+                    </li>
+                </ul>
+            </form>
+            <img class="shape" src="./img/end-line.png" alt="">
         </div>
-    </div>
+
+    </section>
```

Comment:
- [ **OPTIONAL** ] I think it would be better to remove the **bottom scrolling** so your page will be static and centers at one position.

- [ ] Kindly make sure you add the `background icons` in the contact section to make it stick to the Figma design. 
#### Figma design
![cff](https://user-images.githubusercontent.com/83042742/173172127-82d22dea-8ff6-4c66-9aba-3ce34036744e.png)

#### Your design
![bs](https://user-images.githubusercontent.com/83042742/173172108-6e2dfcef-af62-4c5d-ae91-f16bc2a40738.png)